spec/models/merchant_spec.rb

```diff
@@ -11,7 +11,9 @@
 
   describe "relationships" do
     it { should have_many :items }
+    it { should have_many(:item_orders).through(:items) }
```

Comment:
Looks like this works, isn't the syntax here normally supposed to be ``it { should have_many(:item_orders) }`` and ``it { should have_many(:items).through(:item_orders)``?